packages/discord-utilities/src/lib/regexes.ts

```diff
@@ -81,9 +81,9 @@ export const ParsedCustomEmoji = /a?:\w{2,32}:\d{17,18}/;
  * @raw `/^(?<animated>a?):(?<name>[^:]+):(?<id>\d{17,19})$/`
```

Comment:
raw is wrong :)

Reply:
Resolved. Also found 2 other raw's that were wrong